path_planning/src/path_planning.cpp

```diff
@@ -337,13 +345,19 @@ class planner {
 				// vel_pub->publish(minjerk_velocity);
 				// acc_pub->publish(minjerk_accel);
 
+				// Set last path to kino nav path
+				last_path_ = kino_nav_path;
+
     			path_size_int64.data = kino_path.size()+1;
 				path_size_pub->publish(path_size_int64);
```

Comment:
creative indenting?

Reply:
There's creative indenting all over this file, I don't think I added any new creative indenting though.

Reply:
Haha fair enough :laughing: